dbtemplates/migrations/0001_initial.py

```diff
@@ -12,7 +12,7 @@ class Migration(migrations.Migration):
         ('sites', '0001_initial'),
     ]
 
-    if django.get_version() >= '1.8':
+    if django.VERSION[:2] >= (1, 8):
```

Comment:
I think the other branch is not necessary since you're dropping compatibility with Django<1.8 anyway.